user-operator/src/main/java/io/strimzi/operator/user/UserOperatorConfig.java

```diff
@@ -71,6 +73,14 @@ public class UserOperatorConfig {
      * Configures the default prefix of user secrets created by the operator
      */
     public static final ConfigParameter<String> SECRET_PREFIX = new ConfigParameter<>("STRIMZI_SECRET_PREFIX", STRING, "", CONFIG_VALUES);
+
+    /**
+     * Configuration parameter to define a regex pattern used for excluding certain labels
+     * from being considered in processing. This parameter allows for specifying a pattern
+     * that labels must not match to be included. If null, no labels are excluded based on the pattern.
+     */
+    public static final ConfigParameter<Pattern> SECRET_LABELS_EXCLUSION_PATTERN = new ConfigParameter<>("STRIMZI_SECRET_LABELS_EXCLUSION_PATTERN", REGEX_PATTERN, null, CONFIG_VALUES);
```

Comment:
Isn't the name different in the proposal?

Reply:
I was wondering if `SECRET_LABELS_EXCLUSION_PATTERN` would be more accurate to avoid confusion that it does not exclude labels from `KafkaUser` but from `Secret`. I am fine with naming it `STRIMZI_LABELS_EXCLUSION_PATTERN` if you prefer that way :)